std/node/_util/_util_promisify.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+//
+// Adapted from Node.js. Copyright Joyent, Inc. and other Node contributors.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a
+// copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to permit
+// persons to whom the Software is furnished to do so, subject to the
+// following conditions:
+//
+// The above copyright notice and this permission notice shall be included
+// in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+// OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN
+// NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+// OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
+// USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+// In addition to being accessible through util.promisify.custom,
+// this symbol is registered globally and can be accessed in any environment as Symbol.for('nodejs.util.promisify.custom')
+const kCustomPromisifiedSymbol = Symbol.for("nodejs.util.promisify.custom");
+// This is an internal Node symbol used by functions returning multiple arguments
+// e.g. ['bytesRead', 'buffer'] for fs.read.
+const kCustomPromisifyArgsSymbol = Symbol.for(
+  "deno.nodejs.util.promisify.customArgs"
+);
+
+class NodeInvalidArgTypeError extends TypeError {
```

Comment:
I think we are not consistent yet in node api errors.
I suggest supporting all error node error codes in a separate place. _`(std/node/errors ?)`_

but not in this PR, just for a reminder, it will be ok to add a todo.

Reply:
I definitely agree the errors are dispersed and not consistent throughout `std/node`.

Reply:
Can you open an issue for tracking errors?

Reply:
@bartlomieju https://github.com/denoland/deno/issues/5972